addons/project_issue/views/project_project_views.xml

```diff
@@ -24,7 +24,7 @@
         <field name="arch" type="xml">
             <field name="use_tasks" position="after">
                 <field name="use_issues"/>
-                <field name="issue_ids" invisible="1"/>
```

Comment:
OK then. Let's remove it for the good. If someone needs it, then he/she can include the field also.